README.md

```diff
@@ -167,13 +167,13 @@ For more information on kubernetes config files, see https://kubernetes.io/docs/
 
 Kubeaudit produces results with three levels of severity:
 
-`Error`: A security issue or invalid kubernetes configuration
-`Warning`: A best practice recommendation
-`Info`: Informational, no action required. This includes results that are [overridden](#override-errors)
+- `Error`: A security issue or invalid kubernetes configuration
+- `Warning`: A best practice recommendation
+- `Info`: Informational, no action required. This includes results that are [overridden](#override-errors)
```

Comment:
This is an unrelated change, just improves the formatting by separating them severity codes by newlines